test/rescue_test.rb

```diff
@@ -15,125 +15,99 @@ def execute(*args)
       end
 
       let :rescued_plan do
-        execution_plan.rescue_from_error.value
+        world.persistence.load_execution_plan(execution_plan.id)
       end
 
-      describe 'of simple skippable action in run phase' do
+      describe 'no auto rescue' do
+        describe 'of simple skippable action in run phase' do
 
-        let :execution_plan do
-          execute(Example::ActionWithSkip, 1, :error_on_run)
-        end
+          let :execution_plan do
+            execute(Example::ActionWithSkip, 1, :error_on_run)
+          end
 
-        it 'suggests skipping the action' do
-          execution_plan.rescue_strategy.must_equal Action::Rescue::Skip
-        end
+          it 'suggests skipping the action' do
+            execution_plan.rescue_strategy.must_equal Action::Rescue::Skip
+          end
 
-        it 'skips the action and continues' do
-          rescued_plan.state.must_equal :stopped
```

Comment:
By default, the world in tests has `auto_rescue` set to false: therefore it doesn't rescue at all: all the auto-rescuing tests were moved to the `auto rescue` block of the test